picturae_importer.py

```diff
@@ -805,14 +807,15 @@ def hide_unwanted_files(self):
         """
 
```

Comment:
updated filename parsing for hide and unhide files functions in picturae_importer